models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -76,23 +79,28 @@ aeif_cond_beta_multisynapse::Parameters_::Parameters_()
   , t_ref_( 0.0 )     // ms
   , g_L( 30.0 )       // nS
   , C_m( 281.0 )      // pF
-  , E_ex( 0.0 )       // mV
-  , E_in( -85.0 )     // mV
   , E_L( -70.6 )      // mV
   , Delta_T( 2.0 )    // mV
   , tau_w( 144.0 )    // ms
   , a( 4.0 )          // nS
   , b( 80.5 )         // pA
   , V_th( -50.4 )     // mV
-  , I_e( 0.0 )        // pA
+  , num_of_receptors( 1 )
+  , I_e( 0.0 ) // pA
   , gsl_error_tol( 1e-6 )
-  , num_of_receptors_( 1 )
   , has_connections_( false )
 {
+  static const double E_rev_def = 0.0;       // E_rev default value
+  static const double taus_rise_def = 2.0;   // taus_rise default value
+  static const double taus_decay_def = 20.0; // taus_decay default value
```

Comment:
good, now the code is much more compact, there is only the initializer list and the body is empty. For now I'm still using the default with one receptor port, but it will be easy to remove it.